terraform/modules/google-cloud/sql/variables.tf

```diff
@@ -28,7 +28,7 @@ variable "database_name" {
 
 variable "database_version" {
   description = "Version of the Cloud SQL database"
-  default     = "POSTGRES_15"
+  default     = "POSTGRES_16"
```

Comment:
This get used for prod too?

Reply:
yes

Reply:
Were we going to announce this upgrade on prod with a maintenance window? I can draft one today.